lib/client.mli

```diff
@@ -1,38 +1,51 @@
 (** Types and functions pertaining to clients in a P2P application *)
 
-(** Clients are defined by a UDP socket bound
-to an address and port, a reference to some state
-whose type is user-defined, a mutex that gets
-locked whenever the client's socket is being used to
-receive a message (sending is non-blocking), or when the
-state is being read from or written to by the server. *)
+(** Represents a node with some state in a peer-to-peer network *)
```

Comment:
Hum, we're introducing a new word: node.
Should it be replaced by `client` to avoid any confusion? Or is it a different thing?

Reply:
Honestly, maybe `node` is the better word than client? Makes the client/server terminology less confusing by not overloading client? Up to you.

Reply:
Ok for `node`!